### ISSUE TITLE
Allowed Image Patterns locked due to organisations policy when no policy exists

### ISSUE THREAD

**Issue by Louis-Tappenden** [rancher-sandbox/rancher-desktop#5821]
### Actual Behavior

The Allowed Image Patterns in locked enabled with the only allowed images as busybox and nginx. It says locked due to organisations policy, however I have been ensured that no policy exists, and no-one else in the company has had this issue 

### Steps to Reproduce

Delete Rancher desktop and related files from mac, reinstall the latest version from github, launch rancher desktop, open preferences

### Result

<img width="767" alt="Screenshot 2023-10-24 at 12 41 58" src="https://github.com/rancher-sandbox/rancher-desktop/assets/144673168/2fd1f3ae-f932-4553-8f7f-5a28595d5db9">

### Expected Behavior

The Allowed Image Patterns to be unlocked

### Rancher Desktop Version

1.10.0

### Rancher Desktop K8s Version

1.27.6

### Which container engine are you using?

moby (docker cli)

### What operating system are you using?

macOS

### Operating System / Build Version

macOS Sonoma 14.0

### What CPU architecture are you using?

arm64 (Apple Silicon)

### Linux only: what package format did you use to install Rancher Desktop?

None

**Comment by s0nea** [comment]
Hi @Louis-Tappenden,

Thank you for reporting the issue!

Could you please check if you can find the deployment and locked profiles on your file-system as explained here: https://docs.rancherdesktop.io/1.10/getting-started/deployment/#profile-format-and-location ?

**Comment by Louis-Tappenden** [comment]
Yes, I found the deployment profiles and was able to manually change it to disabled allow images. It is still locked when I go directly through the application, but I can change it in vs code through the locked plist file

**Comment by s0nea** [comment]
But this sounds like it works as designed:

- "Defaults" provide preference values that are applied on first run (or after a factory reset).
- "Locked" settings allow an administrator to pin preference values.

If you adapt the files, you'll have to restart Rancher Desktop to apply the changes.

I'm still wondering how you got into the situation. Did you run the test suite or something?
